api/helpers/gamestate/moves/scuttle/validate.js

```diff
@@ -0,0 +1,73 @@
+const GamePhase = require('../../../../../utils/GamePhase.json');
+
+module.exports = {
+  friendlyName: 'Validate request to scuttle',
+
+  description: 'Verifies whether a request to make scuttle is legal, throwing explanatory error if not.',
+
+  inputs: {
+    currentState: {
+      type: 'ref',
+      descriptions: 'Object containing the current game state',
+      required: true,
+    },
+    /**
+     * @param { Object } requestedMove - Object describing the request to scuttle
+     * @param { String } requestedMove.cardId - Card Played for scuttle
+     * @param { String } requestedMove.targetId - Card Targeted for scuttle
+     * @param { MoveType.SCUTTLE } requestedMove.moveType - Specifies that this a scuttle
+     */
+    requestedMove: {
+      type: 'ref',
+      description: 'Object containing data needed for current move',
+      required: true,
+    },
+    playedBy: {
+      type: 'number',
+      description: 'Player number of player requesting move',
+      required: true,
+    },
+  },
+  sync: true,
+  fn: ({ requestedMove, currentState, playedBy }, exits) => {
+    try {
+      const player = playedBy ? currentState.p1 : currentState.p0;
+      const opponent = playedBy ? currentState.p0 : currentState.p1;
+
+      const cardPlayed = player.hand.find(({ id }) => id === requestedMove.cardId);
+      const targetCard = opponent.points.find(({ id }) => id === requestedMove.targetId);
+
+      if (currentState.phase !== GamePhase.MAIN) {
+        throw new Error(`Can only play points in main phase, not ${currentState.phase}`);
+      }
+
+      if (!cardPlayed) {
+        throw new Error('game.snackbar.global.playFromHand');
+      }
+
+      if (cardPlayed.isFrozen) {
+        throw new Error('game.snackbar.global.cardFrozen');
+      }
+
+      if (currentState.turn % 2 !== playedBy) {
+        throw new Error('game.snackbar.global.notYourTurn');
+      }
+
+      if (cardPlayed.rank > 10) {
+        throw new Error('game.snackbar.points.numberOnlyForPoints');
+      }
+
+      if (!targetCard) {
+        throw new Error('game.snackbar.scuttle.mustTargetPointCard');
+      }
+
+      if (cardPlayed.rank < targetCard.rank || (cardPlayed.rank === targetCard.rank && cardPlayed.suit < targetCard.suit)) {
```

Comment:
Let's break this into newlines for linter line length's sake

Reply:
Maybe even capture the chunks in their own variables
```suggestion
      const lowerRank = cardPlayed.rank < targetCard.rank;
      const sameRankLowerSuit = cardPlayed.rank === targetCard.rank && cardPlayed.suit < targetCard.suit;
      if (lowerRank || sameRankLowerSuit) {
```